sample_data_from_verso/data/config/lookupConfig.json

```diff
@@ -0,0 +1,890 @@
+[
+  {
+    "label": "AGROVOC (QA)",
+    "uri": "urn:ld4p:qa:agrovoc",
```

Comment:
Following up on our conversation yesterday with @ndushay, the `uri` be changed to `id` so that profiles with these fields will validate correctly.

Reply:
per discussion with Josh, this should validate as a URI.

Reply:
Yes, Naomi and I talked more yesterday and discovered that `urn:...` is a valid uri.